src/PresentationalComponents/WithPermission/WithPermission.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { usePermissions } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
+import { NotAuthorized } from '@redhat-cloud-services/frontend-components/NotAuthorized';
+
+const WithPermission = ({ children, requiredPermissions = [] }) => {
+    const { hasAccess, isLoading } = usePermissions('patch', requiredPermissions);
```

Comment:
`usePermissions` can be a bit wonky at times, i think using the `usePermissionWithContext` with the RBACProvider setup in the App/Entry.js(somewhere above) would be better.